### ISSUE TITLE
The spec and explainer seem to disagree on what "Cumulative Layout Shift" means

### ISSUE THREAD

**Issue by dholbert** [WICG/layout-instability#49]
Minor issue I noticed when reviewing the [explainer](https://github.com/WICG/layout-instability/blob/master/README.md) / vs. [spec](https://wicg.github.io/layout-instability/):

 - The explainer goes into some detail to define and differentiate the **document cumulative layout shift** vs. the **cumulative layout shift** (the latter of which is computed by performing an aggregating operation across all `iframe`s in the document).

 - However: the spec only mentions the latter term ("cumulative layout shift"), and it uses it with a meaning that seems closer to the other term (i.e. its usage doesn't have any cross-`iframe` aggregation).

To put it another way: when the spec says "Cumulative Layout Shift" (and `cumulativeLayoutShiftScore` in the usage example), it's using it in a sense that corresponds to what the explainer defines as "**Document** Cumulative Layout Shift".

**Comment by skobes** [comment]
This is fixed
